acceptance/openstack/ecs/v1/helpers.go

```diff
@@ -0,0 +1,95 @@
+package v1
+
+import (
+	"testing"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/tools"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/ecs/v1/cloudservers"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func getCloudServerCreateOpts(t *testing.T) cloudservers.CreateOpts {
+	prefix := "ecs-"
+	ecsName := tools.RandomString(prefix, 3)
+
+	az := clients.EnvOS.GetEnv("AVAILABILITY_ZONE")
+	if az == "" {
+		az = "eu-de-01"
+	}
+
+	vpcID := clients.EnvOS.GetEnv("VPC_ID")
+	subnetID := clients.EnvOS.GetEnv("NETWORK_ID")
+	imageID := clients.EnvOS.GetEnv("IMAGE_ID")
+	flavorID := clients.EnvOS.GetEnv("FLAVOR_ID")
+	keyPairName := clients.EnvOS.GetEnv("KEYPAIR_NAME")
```

Comment:
You can create those resources or use some hardcoded values (e.g. for flavor and image), so it will run in CI.